docs/Troubleshooting.md

```diff
@@ -1,13 +1,15 @@
 # Troubleshooting
 
-### `this.context.router` is `undefined`
+### `this.props.router` is `undefined`
```

Comment:
Probably a rephrasing is best. I do think it's relevant, in case someone forgets to wrap their component.